corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -247,12 +282,50 @@ hqDefine('app_manager/js/app_manager.js', function () {
                 $form.find("input[name='name']").val(action === "update" ? "Followup" : "Survey");
                 if (!$form.data('clicked')) {
                     $form.data('clicked', 'true');
-                    $a.find(".fa-plus").removeClass("fa-plus").addClass("fa fa-refresh icon-spin");
+                    $a.find(".fa-plus").removeClass("fa-plus").addClass("fa fa-refresh fa-spin");
                     $form.submit();
                 }
             });
         }
 
+        if (COMMCAREHQ.toggleEnabled('APP_MANAGER_V2')) {
+
+            $('#js-add-new-item').popover({
+                title: "Add",
+                container: 'body',
+                content: function () {
+                    return $('#js-popover-template-add-item-content').text();
+                },
+                html: true,
+                trigger: 'manual',
+                placement: 'right',
+                template: $('#js-popover-template-add-item').text()
+            }).one('shown.bs.popover', function () {
+                var pop = this;
+                $('.popover-additem').on('click', function (e) {
+                    $(pop).popover('hide');
+                    var dataType = $(e.target).closest('button').data('type');
+                    $('#new-module-type').val(dataType);
+                    var form = $('#new-module-form');
+                    if (!form.data('clicked')) {
+                        form.data('clicked', 'true');
+                        $('.new-module-icon').removeClass().addClass("fa fa-refresh fa-spin");
+                        form.submit();
+                    }
+                });
+            }).on('click', function (e) {
+                e.preventDefault();
+                $(this).popover('show');
+            });
+
+            $('body').click(function (event) {
```

Comment:
This logic seems pretty similar to using `focus` for the popover trigger, curious if that would have worked instead.